Sources/CartonKit/Model/Template.swift

```diff
@@ -30,11 +30,12 @@ public enum Templates: String, CaseIterable {
 
 public protocol Template {
```

Comment:
While we're at it, how's about renaming this protocol to `TemplateProtocol` so that `enum Templates` could become `enum Template`? This would reduce any potential confusion.

Reply:
Another option is keeping `protocol Template` as is, but renaming `enum Template` to `enum TemplateName`.